scripts/init_install.sh

```diff
@@ -5,21 +5,13 @@ CELES_HOME=".celestia-app"
 MONIKER="validator"
 ARCHIVE_NAME="payload.tar.gz"
 
-export DEBIAN_FRONTEND=noninteractive 
-
-export SEEN_LIMIT="74" # don't forget to change this!!!
-export INCLUSION_DELAY="45"
+export DEBIAN_FRONTEND=noninteractive
 
 apt update -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"
 
 export DEBIAN_FRONTEND=noninteractive
 apt-get update -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"
-apt-get install git build-essential ufw curl jq chrony snapd --yes -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"
```

Comment:
I guess we can delete ufw as it's not enabled by default so no need to allow ports using it